compute/etc/sql_exporter/compute_logical_snapshots_bytes.libsonnet

```diff
@@ -0,0 +1,17 @@
+local neon = import 'neon.libsonnet';
+
+local pg_ls_logicalsnapdir = importstr 'sql_exporter/compute_logical_snapshots_bytes.16.sql';
+local pg_ls_dir = importstr 'sql_exporter/compute_logical_snapshots_bytes.sql';
+
+{
+  metric_name: 'compute_logical_snapshots_bytes',
+  type: 'gauge',
+  help: 'Size of the pg_logical/snapshots directory, not including temporary files',
+  key_labels: [
+    'timeline_id',
+  ],
+  values: [
+    'logical_snapshots_bytes',
+  ],
+  query: if neon.PG_MAJORVERSION_NUM < 16 then pg_ls_dir else pg_ls_logicalsnapdir,
```

Comment:
I don't quite understand this syntax. what is this query supposed to do?

also, from docs: `pg_ls_logicalsnapdir` was introduced in v15
https://www.postgresql.org/docs/15/functions-admin.html

is there a reason why we use it from 16 only?

Reply:
No, I just thought it was a 16 only function for some reason. Thanks for catching that. I will send a new revision. The syntax says if the major version of postgres is less than 16, use the query which uses pg_ls_dir. Otherwise, use the pg_ls_logicalsnapdir query.